### PR TITLE
Report repo URL

### PR DESCRIPTION
**Pull request by RunsFor**
Patch adds tarantool repository URL to feedback report

### CODE REVIEW

File: src/box/lua/feedback_daemon.lua [hunk 1]
```diff
@@ -217,6 +217,54 @@ local function fill_in_schema_stats(features)
...
+    local content, err = read_first_file('/etc/apt/sources.list.d/tarantool_*.list')
+    if not content then
+        return nil, err
+    end
+
+    return content:match('deb ([^ ]*)')
+end
+
+local function extract_repo_url_yum()
+    local content, err = read_first_file('/etc/yum.repos.d/tarantool_*.repo')
```
Comment (Totktonada) [inline src/box/lua/feedback_daemon.lua:254]: Will miss a repo with dash, like `tarantool-2.5.repo`.

https://github.com/tarantool/doc/issues/1492#issuecomment-685740902

**Review by LeonidVas — COMMENTED** [review]
Hi! Thank you for the patchset.
Generally OK, but I have several minor questions/comments:
- Decorate a commit title (Make test not fail on subsequent runs) according to https://github.com/tarantool/tarantool/wiki/Code-review-procedure#commit-message (test:...).
- What "subsequent runs" do you mean? The test-run clean/remove database after rerun, as I recall.
- Put a period at the end of the sentence (Currently only...) from commit message.

**Comment by RunsFor** [comment]
Hi! I adjusted commit messages, thanks for saying.

> * What "subsequent runs" do you mean? The test-run clean/remove database after rerun, as I recall.

I execute this test on my machine simply by running a script without test-run. So, if I run the script second time, it failed due all spaces and indexes already exist. Is this kind of test executing is not expected?

**Comment by Totktonada** [comment]
> I execute this test on my machine simply by running a script without test-run. So, if I run the script second time, it failes due all spaces and indexes already exist. Is this kind of test executing is not expected?

I often use this way too during debugging and I think it is good property for the test to be ready for that.

File: src/box/lua/feedback_daemon.lua (hunk 1 above)
Comment (RunsFor) [reply src/box/lua/feedback_daemon.lua]: Thanks, I've expanded wildcard to match `tarantool-2.5.repo` as well

**Review by LeonidVas — APPROVED** [review]
LGTM.

**Comment by LeonidVas** [comment]
> > I execute this test on my machine simply by running a script without test-run. So, if I run the script second time, it failes due all spaces and indexes already exist. Is this kind of test executing is not expected?
> 
> I often use this way too during debugging and I think it is good property for the test to be ready for that.

If @Totktonada does so. I don't mind)

**Pull request merged**